.github/workflows/run_tests.yml

```
@@ -118,6 +118,16 @@ jobs:
       - name: Run Pylint
         run: make check-lint
 
+      # Run isort/black style check
+      - name: Run isort
+        run: isort --check-only --profile black ./smartsim ./tests
+
+      # Run isort/black style check
+      - name: Run black
+        run: |
+          black ./smartsim/version.py
+          black --check ./smartsim ./tests
```

Super small request here, I'm not sure that I like the CI is making changes to the code that we generate, especially since those changes will not be made in the release CI.

Can we just have black ignore that file instead?
```suggestion
          black --exclude smartsim/version.py --check ./smartsim ./tests
```

---

that is a way better idea!!